examples/sdk-snapgrid/src/App.css

```
@@ -0,0 +1,1035 @@
+  --glass-border:   rgba(255, 255, 255, 0.08);
+
+  --radius-sm:      8px;
+  --radius-md:      14px;
+  --radius-lg:      20px;
+  --radius-xl:      28px;
+
+  --shadow-card:    0 4px 24px rgba(0,0,0,0.5), 0 1px 3px rgba(0,0,0,0.3);
+  --shadow-hover:   0 12px 48px rgba(0,0,0,0.65), 0 2px 8px rgba(0,0,0,0.4);
+  --shadow-modal:   0 32px 96px rgba(0,0,0,0.8), 0 8px 32px rgba(0,0,0,0.5);
+
+  --font-display:   'Fraunces', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)